setup.py

```diff
@@ -80,7 +80,7 @@
     ],
     package_dir={"": "src"},
     packages=find_packages("src"),
-    python_requires="~=3.7.0",
+    python_requires=">=3.7.0",
```

Comment:
One could even drop the patch version and write `>=3.7`

Reply:
Done in d74a930.